### PR TITLE
Django 1.9 support

### PR DESCRIPTION
**Pull request by kennydude**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -159,8 +152,41 @@ def customer_saved(sender, action=None, instance=None, **kwargs):
...
+      <td>1.5</td>
+      <td>0.0.1</td>
+    </tr>
+    <tr>
+      <td>1.6</td>
+      <td>0.0.3</td>
+    </tr>
+    <tr>
+      <td>1.7</td>
+      <td>0.0.4</td>
```
Comment (scott-w) [inline README.md]: We're missing Django 1.8/1.9 compatibility lines here.

File: django_pgviews/apps.py [hunk 2]
```diff
@@ -24,6 +22,7 @@ def sync_pgviews(self, sender, app_config, **kwargs):
         
         if self.counter == total:
             log.info('All applications have migrated, time to sync')
+            from .models import ViewSyncer
```
Comment (scott-w) [inline django_pgviews/apps.py]: Add a comment here to just say we're preventing circular imports (or whatever the reason is for this).

File: django_pgviews/db/__init__.py [hunk 3]
```diff
@@ -14,5 +14,5 @@ def get_fields_by_name(model_cls, *field_names):
     """
     if '*' in field_names:
         return dict((field.name, field) for field in model_cls._meta.fields)
-    return dict((field_name, model_cls._meta.get_field_by_name(field_name)[0])
+    return dict((field_name, model_cls._meta.get_field(field_name))
```
Comment (scott-w) [inline django_pgviews/db/__init__.py]: Is this backwards compatible?

Comment (kennydude) [inline django_pgviews/db/__init__.py]: I'm not sure but it flags up with it being removed in Django 1.10

File: django_pgviews/management/commands/sync_pgviews.py [hunk 4]
```diff
@@ -11,22 +11,22 @@
 log = logging.getLogger('django_pgviews.sync_pgviews')
 
 
-class Command(NoArgsCommand):
+class Command(BaseCommand):
     help = """Create/update Postgres views for all installed apps."""
```
Comment (scott-w) [inline django_pgviews/management/commands/sync_pgviews.py]: We should check this for backwards compatibility and note it in the README

File: django_pgviews/db/__init__.py (hunk 3 above)
Comment (scott-w) [inline django_pgviews/db/__init__.py]: It's just in case we need to note in the version compatibility for Django.

Comment (kennydude) [inline django_pgviews/db/__init__.py]: Works in 1.8 as well. Just added a circle config to run it on 1.8 as well

**Pull request merged**